typescript/conditions.d.ts

```diff
@@ -5,8 +5,16 @@ export interface ConditionalPredicateBase {
 export type ConditionalPredicateNumeric = ConditionalPredicateBase & {
   type: 'NUMERIC'
   operator: '===' | '!==' | '>' | '>=' | '<' | '<='
-  value: number
-}
+} & (
+    | {
+        compareWith?: 'VALUE'
+        value: number
+      }
+    | {
+        compareWith: 'ELEMENT'
+        value: string
```

Comment:
```suggestion
        elementId: string
```

Reply:
`elementId` is alreadyh a property on this object

Reply:
```suggestion
        compareWithElementId: string
```